core/src/main/java/io/aiven/klaw/model/SyncSchemaUpdates.java

```diff
@@ -6,6 +6,7 @@
 @Data
 public class SyncSchemaUpdates {
   private List<String> topicList;
+  private List<String> schemaRemovalList; // A list of schema to be removed from Klaw metadata
```

Comment:
- typo
- wondering if they are topic names or schema versions ?

Reply:
They are topicnames will update that to specify it exactly